Dockerfile

```diff
@@ -17,9 +17,13 @@ CMD ["/usr/bin/java", "-jar", "/ergo.jar"]
 
 FROM openjdk:10-jre-slim
 LABEL maintainer="Andrey Andreev <andyceo@yandex.ru> (@andyceo)"
-COPY --from=builder /ergo.jar /ergo.jar
+RUN adduser --disabled-password --home /var/lib/ergo --uid 9052 --gecos "ErgoPlatform" ergo && \
```

Comment:
Home dir /var/lib/ergo not used further. Think better use folderless user, but this behaviour require node config changes

Reply:
/home/ergo